### PR TITLE
Format using black

### PR DESCRIPTION
**Pull request by memeplex**
* Formatted everything using black
* Configured pre-commit hooks for flake8 and black
* Listed development dependencies (flake8, black, pre-commit)

### CODE REVIEW

**Comment by memeplex** [comment]
@tzulberti-jampp I'm starting with this one as a first experiment since there are no big pending PRs against gcd. I barely touched the output of black, it has worked great out-of-the-box! Pre-commit hooks worked great also with https://github.com/pre-commit/pre-commit, really simple to install and configure.

**Comment by memeplex** [comment]
With pre-commit you can even check your hooks without commiting:

image

File: setup.cfg [hunk 1]
```diff
@@ -1,5 +1,4 @@
 [flake8]
-ignore = E306, W504
-
-[yapf]
-split_before_named_assigns = false
\ No newline at end of file
+max-line-length = 88
```
Comment (tzulberti-jampp) [inline setup.cfg]: On tech-standardization is mention that the black configuration should allow lines for 120 chars. Check this is the black config we should be using: https://github.com/jampp/cookiecutter-python/blob/develop/%7B%7Bcookiecutter.project_slug%7D%7D/pyproject.toml

We could remove the `skip-*` if you want from the config

Comment (memeplex) [reply setup.cfg]: If tomorrow you're still ok with what we talked over the phone this morning, I'll be marking this as resolved and requesting your approval again.

**Pull request merged**